subsys/bluetooth/controller/hci/hci.c

```diff
@@ -2499,8 +2502,15 @@ static void le_advertising_report(struct pdu_data *pdu_data, u8_t *b,
 #endif /* CONFIG_BT_CTLR_EXT_SCAN_FP */
 	s8_t *prssi;
 
-	extra = &b[offsetof(struct node_rx_pdu, pdu) +
-		   offsetof(struct pdu_adv, payload) + adv->len];
+#if defined(CONFIG_BT_LL_SW_SPLIT)
+	struct node_rx_pdu *node_rx;
+
+	node_rx = (struct node_rx_pdu *)b;
+	extra = (u8_t *) &(node_rx->hdr.rx_ftr.param);
+#else
+	extra = (u8_t *) &b[offsetof(struct radio_pdu_node_rx, pdu_data) +
```

Comment:
no space in between ```(u8_t *) &(```

Reply:
Thanks for pointing this out, I have fixed this in several places.